docs/cli/usage.md

```diff
@@ -1,24 +1,38 @@
 # Usage
 
-This assumes that you are using the Docker Container.
+## Running neuroscout-cli
+
 Docker commands for _neuroscout-cli_ will always being with:
 
     docker run -it --rm
 
-Assuming you've already created an analysis on neuroscout.org, and have its analysis id (e.g.: `5xH93`), you can run it in one line:
+Or if you are using Singularity:
+
+    singularity run --cleanenv
+
+Assuming you've already created an analysis on neuroscout.org, and have its analysis id (e.g.: `5xH93`), you can run it in one line using docker:
 
     docker run -it --rm neuroscout/neuroscout-cli run /out 5xH93
 
+or singularity:
+
+    singularity run --cleanenv neuroscout-cli-<version>.simg /out 5xH93
+
 Neuroscout will download the necessary images, analysis bundle, and fit your model.
 
```

Comment:
May be useful to also explicitly cover how to run on a SLURM system, as they do i fmriprep:

https://fmriprep.org/en/stable/singularity.html#running-singularity-on-a-slurm-system